akka-http2-support/src/test/scala/akka/http/impl/engine/http2/TelemetrySpiSpec.scala

```diff
@@ -34,6 +34,8 @@ import org.scalatest.concurrent.ScalaFutures
 import java.util.UUID
 import scala.concurrent.Future
 import scala.concurrent.duration._
+import scala.util.Failure
```

Comment:
Are these changes related to the PR?

Reply:
Mostly not, but some are. 
In this file I renamed a value but also had to reorder some expectations since the shutdown sequence varied a bit.